samcli/lib/providers/cfn_api_provider.py

```diff
@@ -382,12 +412,15 @@ def _get_integration_function_name(integration):
 
         if integration and isinstance(integration, dict):
             # Integration must be "aws_proxy" otherwise we don't care about it
-            return LambdaUri.get_function_name(integration.get("Uri"))
+            uri: str = cast(str, integration.get("Uri"))
+            return LambdaUri.get_function_name(uri)
```

Comment:
why did we split this in to two lines? did mypy complain?

Reply:
Yes mypy complaint, `dict.get()` returns `Any`, I made it into two lines, one for casting and one for usage so that the lines are more readable.